templates/api_info.mustache

```diff
@@ -0,0 +1,9 @@
+{{#appName}}
```

Comment:
Overriding this template is the primary change here.  The original (default) version version cane be found here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/ruby-client/api_info.mustache